estate/models/estate_property.py

```diff
@@ -0,0 +1,98 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate"
+    _sql_constraints = [
+        ('check_expected_price', 'CHECK(expected_price > 0)', 'Expected price must be strictly positve'),
+        ('check_selling_price', 'CHECK(selling_price >= 0)', 'Selling price must be positive')
+    ]
+    _order = 'id desc'
+
+    name = fields.Char('Name', required=True)
+    description = fields.Char('Description')
+    post_code = fields.Char('Post Code')
+    state = fields.Selection(
+        string="State",
+        selection=[('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accepted'), ('sold', 'Sold'), ('canceled', 'Canceled')],
+        help="Estate current state",
+        required=True,
+        default='new',
+        copy=False
+    )
```

Comment:
I prefer this format more and its the common one for selection list. Also, we use snack_case for the selection values
```suggestion
        selection=[
            ('new', 'New'), 
            ('offer_received', 'Offer Received'), 
            ('offer_accepted', 'Offer Accepted'), 
            ('sold', 'Sold'), 
            ('canceled', 'Canceled')],
        help="Estate current state", required=True, default='new', copy=False)
```